pegasus/router.rb

```diff
@@ -599,8 +601,9 @@ def social_metadata
       metadata
     end
 
-    def view(uri, locals={})
+    def view(uri, locals={}, html_only=false)
       path = resolve_template('views', settings.template_extnames, uri.to_s)
+      raise "View `#{uri}` must be .html. #{File.extname(path)}" if html_only && File.extname(path) != ".html"
       raise "View '#{uri}' not found." unless path
```

Comment:
This leaves us vulnerable to templates which follow the common Rails naming convention of `.haml.html` I'd recommend instead doing something like:

```suggestion
      raise "View '#{uri}' not found." unless path

      if html_only
        template_is_only_html = MultipleExtnameFileUtils.all_extnames(path).all?('.html')
        raise "View `#{uri}` must be .html. #{File.extname(path)}" unless template_is_only_html
      end
```